conf/standalone.conf

```
@@ -184,6 +184,10 @@ maxTopicsPerNamespace=0
 # 'is_allow_auto_update_schema' of namespace policy.
 isAllowAutoUpdateSchemaEnabled=true
 
+# Override the schema registry used by pulsar with a custom implementation. If this config is not provided,
+# the default schema registry will be used.
+schemaRegistryClassName=
```

```suggestion
schemaRegistryClassName=org.apache.pulsar.broker.service.schema.SchemaRegistryServiceImpl
```

And please also help add the config option to broker.conf

---

updated it